client/src/controllers/TooltipController.test.js

```diff
@@ -133,4 +133,26 @@ describe('TooltipController', () => {
 
     expect(tooltip.textContent).toEqual('NEW content!');
   });
+  it('should support passing the offset value', async () => {
+    document.body.innerHTML = `
+    <section>
+      <button
+        id="button"
+        type="button"
+        data-controller="w-tooltip"
+        data-w-tooltip-offset-value="[10, 20]"
+      >
+        CONTENT
+      </button>
+    </section`;
```

Comment:
```suggestion
    </section>`;
```

This is not resolved, the new code added is missing the closing bracket. thanks for fixing the one above though.